recipes/dearpygui/patch_windows_freetype.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/setup.py b/setup.py
+index c35ef0dc..8ffafb7f 100644
+--- a/setup.py
++++ b/setup.py
+@@ -65,10 +65,11 @@ def run(self):
+         return
+ 
+     if get_platform() == "Windows":
++        prefix = os.environ.get('PREFIX', '')
+         command = [r'set PATH="C:\Program Files (x86)\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin";"C:\Program Files (x86)\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin";"C:\Program Files (x86)\Microsoft Visual Studio\2022\Professional\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin";%PATH% && ']
```

Comment:
you should also patch this out. this change to the path doesn't need to be done.